### DISCUSSION TITLE
File permissions: readable for everyone

### DISCUSSION THREAD

**Discussion by sogeil** [ltsp/ltsp#861] [Q&A]
Hello!

Under /usr/bin I have a few bash scripts that I use with the Thunar file browser.
Advantage of /usr/bin: the entire path to the script does not have to be entered.

"Reading" of the file must be allowed for all users.

However, this does not work with LTSP: the rights for owner and group are correct in the /usr/bin of the clients, but not for "Other".

How can I change this?
And if that's not possible: where is the best place to store the scripts so that every user can use them with LTSP?

Greeting,
Olaf

**Comment by alkisg** [comment]
Hi Olaf, what is the output of this command on your server and on your clients?

ls -ld /usr/bin

They're supposed to be the same, LTSP doesn't change them.
Although, the correct place to put local applications is in /usr/local/bin.
